tests/hooks/test-opencode-hook-bridge.sh

```
@@ -0,0 +1,980 @@
+# check turns that from "the implementer's inline reasoning" into "an
+# executable, reproduced fact" (the exact gap the S1 security
+# re-assessment flagged). Deliberately NOT `record_fail`: folding an
+# open disposition question into the hard-fail/exit-1 count would make
+# this suite block on a decision this file cannot make. Deliberately
+# NOT `record_skip` either: the behavior WAS exercised, not bypassed.
+# Printed loudly in its own summary block below so it cannot be missed.
+record_finding() {
+    findings=$((findings + 1))
+    finding_notes+=("$1")
+    printf 'FINDING  %s\n' "$1"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)